go/client/rpc.go

```diff
@@ -24,7 +24,7 @@ func getSocket(g *libkb.GlobalContext, clearError bool) (xp rpc.Transporter, err
 	var isNew bool
 	_, xp, isNew, err = g.GetSocket(clearError)
 	if err == nil && isNew {
-		introduceMyself(g, xp)
+		_ = introduceMyself(g, xp)
```

Comment:
why not return this error?

Reply:
Good question, I don't remember having a reason.  I'll change it.